test/utils.js

```diff
@@ -3,7 +3,7 @@ import { RuleTester as EslintRuleTester } from 'eslint';
 import { test } from 'mocha';
 
 /* pattern taken from eslint-plugin-import */
-export function addFilenameOption(testCase) {
+export function addFilenameOption (testCase) {
```

Comment:
A linter made you _add_ a whitespace here?! That's odd 🤔

Reply:
Yeah, that's a rule in `eslint-plugin-standard`. Might change the style later though.